content/ghc-9.12-cabal-3.14/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "GHC 9.12 & cabal 3.14 releases"
+date = 2024-12-22
+[taxonomies]
+authors = ["Hécate"]
+categories = ["Cabal", "GHC"]
+tags = ["Release"]
++++
+
+To conclude the year 2024, the GHC and Cabal teams are happy to announce the release of GHC 9.12 and cabal 3.14.
+
+<!-- more -->
+
+
+Here are some highlights:
+
+## GHC 9.12
+
+### Language Extensions
+
+This release brings its lot of new and exciting extensions:
+* [Multiline string literals][MultilineStrings] to spread a piece of text over several lines;
```

Comment:
Not like it was impossible before, so probably needs to be unpacked for readers with more details. CC @brandonchinn178

Reply:
Sure  something like "to write string literals with newlines kept verbatim, enabling multiline strings without the need for string gaps and literal `\n` characters"?

Reply:
I considered commenting on that, but decided that a proper description might swamp the rest of the release announcement.

Reply:
I think it's exactly at the right level of detail now.

Reply:
The art of release announcement is to be both precise and concise. If we leave it as is, someone can easily quote it as a proof that Haskell had no multiline literals before 2024 and one will have hard time to prove otherwise. I think @brandonchinn178's suggestion is good.